<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>record_replay_dialog</class>
 <widget class="QWidget" name="record_replay_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>record&amp;replay</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_saveAs">
         <property name="text">
          <string>saveAs</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_clear">
         <property name="toolTip">
          <string>remove all operations.</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_cut">
         <property name="toolTip">
          <string>remove the remainning operations.</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="text">
          <string>cut</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_load">
         <property name="text">
          <string>load</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_save">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton_pause">
            <property name="text">
             <string>pause</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_play">
            <property name="text">
             <string>play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_record">
            <property name="text">
             <string>record</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_observer">
         <property name="toolTip">
          <string>send message to observer.</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="text">
          <string>observer</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="current_total">
       <property name="text">
        <string>current/total</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="file_path">
     <property name="text">
      <string>file:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>author: Simba,  email: lisiwangcg@gmail.com</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
